DTS-STN/secure-client-hub
# 134855 react18 next13

## [ADO-134855](https://dev.azure.com/VP-BD/DECD/_workitems/edit/134855)

Fixed [AB#134855](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/134855)

### Changelog - "fix:" for bug fixes, "feat:" for features. Read more about Conventional Commits at https://www.conventionalcommits.org/en/v1.0.0/#summary

### Description of proposed changes:
Updated to React 18 and Next 13.

### What to test for/How to test
1. Pull in branch
2. Delete your `node_modules/` folder by running `rm -rf node_modules/` and then delete your package-lock (read additional notes for explanation)
3. Type `npm i`
4. Type `npm run dev`
5. Ensure application runs as expected

### Additional Notes
After upgrading to React 18, there was a stray dependency (react-dom 17.0.2) that was left behind that messes up things on our app's side when installing the new version of DS. The temporary solution for this is to nuke node_modules and package-lock and then install. The dependency has been removed from DS though and this fix will be in their next release.
